MerlinAU.sh

```diff
@@ -1972,7 +2022,9 @@ _CheckNewUpdateFirmwareNotification_()
            fwNewUpdateNotificationDate="$(date +"$FW_UpdateNotificationDateFormat")"
            Update_Custom_Settings FW_New_Update_Notification_Vers "$fwNewUpdateNotificationVers"
            Update_Custom_Settings FW_New_Update_Notification_Date "$fwNewUpdateNotificationDate"
+           if [ ! "$isInteractive" ]; then
```

Comment:
IMO, the email notification when a new F/W Update becomes available should be sent whether the script is running interactively or not because it becomes a record of when the user was notified of the new F/W version & the email contains the number of days the update is postponed. This is important information to have readily available in your email so you can easily check without opening an SSH session and logging into the router.